api_inference/inference-api.ipynb

```diff
@@ -17,75 +17,90 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install openai"
+    "!pip install huggingface_hub"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 7,
    "id": "709fe89b",
    "metadata": {},
    "outputs": [],
    "source": [
-    "import huggingface_hub\n",
-    "from openai import OpenAI"
+    "from huggingface_hub import InferenceClient, get_token, login"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "37aedded",
+   "metadata": {},
+   "source": [
+    "Please, ensure you are logged in to Hugging Face or run the following cell:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6649919b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "login()"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "b2084476",
    "metadata": {},
    "source": [
-    "We create a client for the Inference API endpoint, passing our HF token. Please, ensure you are logged in to Hugging Face."
+    "We initialize a client for the Inference API endpoint and pass our HF token.\n"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 5,
    "id": "c9914c23",
    "metadata": {},
    "outputs": [],
    "source": [
-    "client = OpenAI(\n",
-    "    base_url=\"https://api-inference.huggingface.co/v1/\",\n",
-    "    api_key=huggingface_hub.get_token(),\n",
-    ")"
+    "hf_token = get_token()\n",
+    "client = InferenceClient(token=hf_token)"
```

Comment:
```suggestion
    "client = InferenceClient()"
```

No need to `get_token` since we use the `InferenceClient` now.

Reply:
true! thanks